src/test/java/cucumber/steps/SearchSteps.java

```diff
@@ -20,11 +20,14 @@
     AppLauncherModalPage appLauncherModalPage;
     List<WebElement> list;
 
-    public SearchSteps(ApiRequestBuilder requestBuilder, ApiResponse apiResponse) {
-        this.requestBuilder = requestBuilder;
-        this.apiResponse = apiResponse;
+    public SearchSteps(final ApiRequestBuilder newRequestBuilder, final ApiResponse newApiResponse) {
```

Comment:
Im not sure if this method has to be in the steps?

Reply:
I'll delete it.